rosdep/python.yaml

```
@@ -7488,6 +7488,19 @@ python3-multimethod-pip:
   ubuntu:
     pip:
       packages: [multimethod]
+python3-munkres:
+  debian:
+    bullseye: [python3-munkres]
+    buster: [python3-munkres]
```

The distro names can be elided since all currently supported Debian distributions package this software.

```suggestion
  debian: [python3-munkres]
```